examples/similarity_search/add_faces.py

```diff
@@ -1,23 +1,35 @@
 import argparse
 from aperturedb import Utils, ParallelLoader
-from facenet import generate_embedding
 import dbinfo
 from CelebADataKaggle import CelebADataKaggle
+from aperturedb.transformers.facenet_pytorch_embeddings import FacenetPyTorchEmbeddings
+from aperturedb.transformers.common_properties import CommonProperties
+from aperturedb.transformers.image_properties import ImageProperties
 
 search_set_name = "similar_celebreties"
 
 
 def main(params):
     utils = Utils.Utils(dbinfo.create_connector())
     utils.remove_descriptorset(search_set_name)
-    utils.add_descriptorset(search_set_name, 512,
-                            metric="L2", engine="FaissFlat")
-
-    dataset = CelebADataKaggle(
-        records_count=params.images_count,
-        embedding_generator=generate_embedding,
-        search_set_name=search_set_name
-    )
+
+    dataset = CelebADataKaggle()
+
+    # Here's a pipeline that adds extra properties to the celebA dataset
+    dataset = CommonProperties(
+        dataset,
+        adb_data_source="kaggle-celebA",
+        adb_main_object="Face",
+        adb_annoted=True)
+
+    # some useful properties for the images
+    dataset = ImageProperties(dataset)
+
+    # Add the embeddings generated through facenet.
+    dataset = FacenetPyTorchEmbeddings(dataset)
```

Comment:
Oh this is repeated because we have this in examples and somewhere else but I am not clear why.
Also, same questuon on creating embeddings if one wants only image props

Reply:
yes repeated in how examples have it, and now the CLI can pick and choose these to build a pipeline.
Any combination is allowed through CI.